Assets/MixedRealityToolkit.SDK/Features/Input/InputSimulationIndicators.cs

```diff
@@ -0,0 +1,112 @@
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// A row of indicator buttons to control input simulation features.
+    /// </summary>
+    [Serializable]
+    public class InputSimulationIndicators : MonoBehaviour
+    {
+        /// <summary>
+        /// Icon component for the left hand.
+        /// </summary>
+        public UnityEngine.UI.Image imageHandLeft = null;
+        /// <summary>
+        /// Icon component for the right hand.
+        /// </summary>
+        public UnityEngine.UI.Image imageHandRight = null;
+
+        public Sprite iconHandActiveLeft = null;
+        public Sprite iconHandActiveRight = null;
+        public Sprite iconHandIdleLeft = null;
+        public Sprite iconHandIdleRight = null;
+        public Sprite iconHandUntrackedLeft = null;
+        public Sprite iconHandUntrackedRight = null;
+
+        private IInputSimulationService inputSimService = null;
+        private IInputSimulationService InputSimService
+        {
+            get
+            {
+                if (inputSimService == null)
+                {
+                    if (MixedRealityServiceRegistry.TryGetService<IMixedRealityInputSystem>(out IMixedRealityInputSystem inputSystem))
+                    {
+                        inputSimService = (inputSystem as IMixedRealityDataProviderAccess).GetDataProvider<IInputSimulationService>();
+                    }
+                }
+                return inputSimService;
+            }
+        }
+
+        /// <inheritdoc />
+        void Update()
+        {
+            if (imageHandLeft)
+            {
+                Sprite iconHandLeft;
+                if (InputSimService.IsSimulatingHandLeft)
+                {
+                    iconHandLeft = iconHandActiveLeft;
+                }
+                else if (InputSimService.HandDataLeft.IsTracked)
+                {
+                    iconHandLeft = iconHandIdleLeft;
+                }
+                else
+                {
+                    iconHandLeft = iconHandUntrackedLeft;
+                }
+
+                imageHandLeft.sprite = iconHandLeft;
+            }
+
+            if (imageHandRight)
+            {
+                Sprite iconHandRight;
+                if (InputSimService.IsSimulatingHandRight)
+                {
+                    iconHandRight = iconHandActiveRight;
+                }
+                else if (InputSimService.HandDataRight.IsTracked)
+                {
+                    iconHandRight = iconHandIdleRight;
+                }
+                else
+                {
+                    iconHandRight = iconHandUntrackedRight;
+                }
+
+                imageHandRight.sprite = iconHandRight;
+            }
+        }
+
+        public void ToggleLeftHand()
```

Comment:
>ToggleLeftHand [](start = 20, length = 14)

Nit: Can you also add doccomments to these four functions (i.e. toggle/reset/*)?

Mainly because ToggleLeftHand toggles the visibility (which doesn't come up as a word in the function name) (otherwise, is it toggling its tracking state/not tracking state? - it's a bit ambiguous)

Same comment around reset (i.e. what is it resetting? visibility state? tracked state? simulation state? all up state?)